more/free-programming-cheatsheets.md

```diff
@@ -128,6 +128,7 @@
 
 ### HTML and CSS
 
+* [Accessibility CheatSheet](https://learn-the-web.algonquindesign.ca/topics/accessibility-cheat-sheet/) - Learn the Web, Algonquin Design (HTML)
```

Comment:
In this case we only need the corporate author, a.k.a. those text where is part of the copyright layer in the footer

```suggestion
* [Accessibility CheatSheet](https://learn-the-web.algonquindesign.ca/topics/accessibility-cheat-sheet/) - Algonquin Design (HTML)
```

Reply:
done as per you suggestions @davorpa